nexus/tests/integration_tests/endpoints.rs

```diff
@@ -378,21 +386,39 @@ pub struct VerifyEndpoint {
 }
 
 /// Describes the visibility of an HTTP resource
+#[derive(Debug)]
 pub enum Visibility {
-    /// All users can see the resource (including unauthenticated or
-    /// unauthorized users)
+    /// Any authenticated user can see the resource, it is not protected by
```

Comment:
Because this is now per-method and not per-endpoint, I think it's no longer `Visibility` but `UnprivilegedAccess` .  I'm not saying you should call it that exactly but `Visibility` seems wrong now, as do the doc comments.  It's not that any authenticated user can _see_ this resource any more.  It's that any authenticated user can do a specific thing on this resource (determined by whatever `AllowedMethod` this thing is hanging off).

Reply:
No longer applies, I went with `UnprivilegedAccess`